spec/models/ontology_spec.rb

```diff
@@ -339,6 +339,26 @@
         expect(ontology.sentences_count).to eq(ontology.sentences.count)
       end
     end
+
+    context 'axioms count' do
+      it 'should be correct' do
+        expect(ontology.axioms.count).to eq(1)
+      end
+
+      it 'should be reflected in the corresponding field' do
+        expect(ontology.axioms_count).to eq(ontology.axioms.count)
```

Comment:
This is not quite the same. On the left hand side, we have `axioms_count` (underscore) which is a column in the database and on the right hand side we have `axioms.count` (dot) which actually sends an sql query to count the rows.